Website/src/items/content/EdgeWebsite/home/speakers/edjones/Page Components/edjones-jss-main-HeroImage-1.yml

```diff
@@ -0,0 +1,34 @@
+﻿---
+ID: "24baadf6-0254-58a0-a0bd-301ef9ca5e7a"
+Parent: "7c0859fa-b53f-56d5-bc7b-1305b857c48f"
+Template: "3ee5e476-e843-51f1-a351-d2c3b198475a"
+Path: "/sitecore/content/EdgeWebsite/home/speakers/edjones/Page Components/edjones-jss-main-HeroImage-1"
+Languages:
+- Language: en
+  Fields:
+  - ID: "b5e02ad9-d56f-4c41-a065-a133db87bdeb"
+    Hint: __Display name
+    Value: "HeroImage-1"
+  Versions:
+  - Version: 1
+    Fields:
+    - ID: "25bed78c-4957-4165-998a-ca1b52f67497"
+      Hint: __Created
+      Value: 20210721T153623Z
+    - ID: "7f1d47cf-4525-5f3e-b8fe-4c6ddbcefa39"
+      Hint: hero
+      Value: |
+        <image alt="Sample Image" mediaid="{DEAF21AB-DEA6-5FEF-8893-4CFEB946819F}" />
```

Comment:
In all the hero image datasource items, should the image come from ContentHub DAM?

Reply:
couldnt add the golf image from DAM
![image](https://user-images.githubusercontent.com/82103130/127044106-edb9bb6b-9daf-4a6f-b046-68434ee40ffa.png)